paypal/express/facade.py

```diff
@@ -37,84 +34,108 @@ def get_paypal_url(basket, shipping_methods, user=None, shipping_address=None,
         currency = getattr(settings, 'PAYPAL_CURRENCY', 'GBP')
     if host is None:
         host = Site.objects.get_current().domain
-    if scheme is None:
-        use_https = getattr(settings, 'PAYPAL_CALLBACK_HTTPS', True)
-        scheme = 'https' if use_https else 'http'
-
-    response_view_name = 'paypal-handle-order' if buyer_pays_on_paypal() else 'paypal-success-response'
-    return_url = '%s://%s%s' % (scheme, host, reverse(response_view_name, kwargs={'basket_id': basket.id}))
-    cancel_url = '%s://%s%s' % (scheme, host, reverse('paypal-cancel-response', kwargs={'basket_id': basket.id}))
-
-    # URL for updating shipping methods - we only use this if we have a set of
-    # shipping methods to choose between.
-    update_url = None
-    if shipping_methods:
-        update_url = '%s://%s%s' % (scheme, host, reverse('paypal-shipping-options', kwargs={'basket_id': basket.id}))
-
-    # Determine whether a shipping address is required
-    no_shipping = False
-    if not basket.is_shipping_required():
-        no_shipping = True
-
-    # Pass a default billing address is there is one.  This means PayPal can
-    # pre-fill the registration form.
+    scheme = getattr(settings, 'PAYPAL_CALLBACK_SCHEME', 'https')
```

Comment:
I thought that it is better to indicate what schema you want to use explicitly (with `PAYPAL_CALLBACK_SCHEME`) instead of indicating that you want to use `https` with boolean.
Means I prefer this:
```python
scheme = getattr(settings, 'PAYPAL_CALLBACK_SCHEME', 'https')
```
instead of this
```python
use_https = getattr(settings, 'PAYPAL_CALLBACK_HTTPS', True)
scheme = 'https' if use_https else 'http'
```

The one con that I can see with my change is that the previous setting will not work anymore, but since there will be so many backward-incompatible changes, I do not think that this is what we should afraid of.